### DISCUSSION TITLE
Is there a way to get pre-timed song/ .osz file from featured artists on lazer's editor?

### DISCUSSION THREAD

**Discussion by Rhythmic-Ocean** [ppy/osu#25514] [Q&A] [answered]
Is there a way to do that on lazer or do we have to manually do it from stable website (like downloading the .osz file from stable website and opening it on lazer and then start mapping it from there)? Would be cool if we had something like featured artist listing just like beatmap listing, but for editor. Sry, if I wasn't supposed to ask this here, first time doing this.

**Comment by peppy** [comment, marked as answer]
You will need to use the website for now.

**Reply by Rhythmic-Ocean** [reply]
ok
